draft-quic-atsss-reqs.mkd

```diff
@@ -424,8 +422,8 @@ If the server is supporting MPTCP natively, the Convert Protocol provides an opt
 
 The MPTCP-based ATSSS discussed in the previous section is specific to TCP,
 obviously. Therefore it does not support non-TCP traffic, such as
-UDP, RTP, QUIC {{QUIC-Deployment}} or IPSec and DTLS VPN services.
```

Comment:
Why removing RTP ?

Reply:
We do already have many examples + I'd like to avoid discussion on particular protocols as there are specific multipath extensions for these protocols. For example, avtcore defines Multipath RTP, etc.

Reply:
I agree that mention RTP doesn't add anything particular but could let to further (out-of-scope) discussions...